tests/Unit/DataStructures/Test_DataVector.py

```diff
@@ -38,5 +41,167 @@ def assignment_test():
         self.assertRaises(RuntimeError, assignment_test)
 
 
+    def test_abs(self):
+        a=DataVector(5,-6)
```

Comment:
We normally have a space between function arguments. E.g. `(5, -6)`. We also have spaces are the `=` too. 

Please apply this throughout the file.